src/moderngl.cpp

```diff
@@ -8254,6 +8254,40 @@ static int parse_blend_func(PyObject * arg, int * value) {
     return 1;
 }
 
+static int parse_blend_func_i(PyObject * arg, int * value) {
```

Comment:
please change this setter to a method. see ctx.enable() / Context_meth_enable for example